README.md

```diff
@@ -68,7 +67,7 @@ Usage of ./ipam-driver:
 For example,
 
 ```
-./ipam-driver --grid-host=192.168.124.200 --wapi-username=cloudadmin --wapi-password=cloudadmin --local-view=global_view --local-network-container="192.168.0.0/24,192.169.0.0/24" --local-prefix-length=24
+./ipam-driver --grid-host=192.168.124.200 --wapi-username=cloudadmin --wapi-password=cloudadmin --local-view=local_view --local-network-container="192.168.0.0/24,192.169.0.0/24" --local-prefix-length=24
```

Comment:
Please include the global view option, too. Also, make the network containers /20 or maybe make one a /20 and the other a /22 just to show how they can be different. The local prefix length could be a /25.